### ISSUE TITLE
xcode 14.3.1 error

### ISSUE THREAD

**Issue by baiyidjp** [relatedcode/ProgressHUD#155]
I use xcode 14.3.1,HUD version is 14.1.0, this code build error, I do not hava ios17

```
		if #available(iOS 17.0, *) {
			imageView.addSymbolEffect(.bounce, options: .repeating)
		}
```

**Comment by CNDEVE** [comment]
the same  xecode version 14.2
if #available(iOS 17.0, *) {
			imageView.addSymbolEffect(.bounce, options: .repeating)
		}
errorMsg: Cannot infer contextual base in reference to member 'bounce , Cannot infer contextual base in reference to member 'repeating', Value of type 'UIImageView' has no member 'addSymbolEffect'

**Comment by baiyidjp** [comment]
> @baiyidjp @CNDEVE In case you're using Xcode 14.3.1 or an earlier version, stick with version 13.8.6.

yes, I  can use 13.8.6
